app/models/application_record.rb

```diff
@@ -23,6 +23,10 @@ class ApplicationRecord < ActiveRecord::Base
           joins(:position)
             .where(positions: { department_id: department_ids })
         }
+  scope :hired_count, lambda {
```

Comment:
Nitpick, but it seems like we should either do:

1. `hire_count` and `interview_count`, or
2. `hired_count` and `interviewed_count`.